src/main/java/bio/terra/cli/command/Notebooks.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.cli.command;
+
+import bio.terra.cli.command.notebooks.Create;
+import picocli.CommandLine;
+
+/**
+ * This class corresponds to the second-level "terra notebook" command. This command is not valid by
+ * itself; it is just a grouping keyword for it sub-commands.
+ */
+@CommandLine.Command(
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.